### PR TITLE
Switch to use code generated by codegen's protoc

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -50,14 +50,6 @@ http_archive(
     ],
 )
 
-### Protoc-gen-validate
-git_repository(
-    name = "com_github_envoyproxy_protoc_gen_validate",
-    commit = "9eff07ddfcb4001aa1aab280648153f46e1a8ddc",
-    remote = "https://github.com/envoyproxy/protoc-gen-validate.git",
-    shallow_since = "1560436592 +0000",
-)
```
Comment (khanhtc1202) [inline WORKSPACE:59]: finally 👋

**Comment by khanhtc1202** [comment]
Here you go 👍 
/approve

File: WORKSPACE (hunk 1 above)
Comment (nghialv) [reply WORKSPACE:59]: 😄

**Comment by nghialv** [comment]
Fixed this PR. Please take a look at this when you have time.
(The CodeGen action returned an error but it is ok to ignore it at this time.)

**Comment by khanhtc1202** [comment]
Nice work
/lgtm

**Pull request merged**